src/schema/rules/sidecars/mri.yaml

```
@@ -37,6 +37,7 @@ MRIHardware:
     MatrixCoilMode: recommended
     CoilCombinationMethod: recommended
     NumberTransmitCoilActiveElements: optional
+    TablePosition: recommended
```

Please note that `recommended` means that the validator will warn if the metadata are not present. I think it should probably be `optional`.

```suggestion
    TablePosition: optional
```

If we want to make it `recommended` if the `chunk` entity is present:

```suggestion
    TablePosition:
      level: optional
      level_addendum: recommended if `chunk` entity is present

MRIChunkPosition:
  selectors:
    - modality == "mri"
    - entities.chunk
    - match(extension, '\.nii(\.gz)?$')
  fields:
    TablePosition:
      level: recommended
      issue:
        code: TABLE_POSITION_RECOMMENDED
        message: |
          TablePosition is RECOMMENDED if the chunk entity is present.
```

---

The use case I have for the TablePosition tag is to find the coordinates of the isocenter in RAS coordinates. The isocenter can then be used for shimming. The TablePosition is therefore not only useful when there are different chunks. If it is optional, then I worry that the information will not be available and we won't have access to the isocenter location. However, I use dcm2niix so since it is implemented there, I believe I won't have issues for the data that I acquire but could run into issues if converted from somewhere else. @effigies you are more aware of the impacts that it might have so let me know what you think.

---

Current definition, "The table position, relative to an implementation-specific reference point,
    often the isocenter." include some vagueness about the reference point, not specific to the isocenter. Is this intentional?

---

I could not 100% percent validate that a table position of 0 means that the patient position affine origin is at isocenter. From my experience, it has always been the case (on Siemens and Philips), but I do not have access to all manufacturers private header information to assume this hypothesis. "relative to an implementation-specific reference point" is the wording used for the [DICOM Table position CT tag](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.15.3.5.html) which is why I chose to include it in the definition, however I would love to be able to drop the vagueness if we can.

---

I am not trying to say it is only useful with multiple chunks, but that it goes from being nice-to-have to potentially-difficult-to-process-without in a multi-chunk context. It is certainly valid to advertise that you require the metadata in order to use your tool.

I'm coming at this from the perspective of validation. We have accumulated many RECOMMENDED fields over the years, and a validator that warns on all of them produces quite a lot of text. I believe we should be judicious about adding new recommendations, and target them to contexts where they are most likely to do good. I do think it's a good thing for tools like dcm2niix to aim to be as complete as possible, to make the dataset as broadly usable as possible, but a validator needs to raise *actionable* warnings and errors. An error should indicate that a dataset is wrong, and a warning should indicate that curators should make a considered decision about whether to address it.

(As an aside, I also think we should bump many existing recommendations down to optional, for the reasons above.)

---

That sounds reasonable :)

---

I'm marking this conversation resolved. @mr-jaemin If you want to further discuss wording, please respond on the main thread.